index.js

```diff
@@ -183,9 +183,23 @@ function init (options) {
   }
 }
 
+// Retrieve a path by specifying a registered alias
+function getAliasPath(alias) {
+    if (typeof alias !== 'string') {
+        throw new Error('[' + alias + ']: is not of type string');
+    }
+
+    if (!moduleAliases[alias]) {
+        throw new Error('[' + alias + ']: alias does not exist');
```

Comment:
I think it's better if the parent function will decide whether or not it should throw in case of not found alias

Reply:
Sorry would you mind clarifying what you mean by parent function? Might be misunderstanding